ipmininet/router/config/utils.py

```diff
@@ -36,3 +36,12 @@ def ip_statement(ip: Union[int, str, IPv6Address, IPv4Address]):
     if not isinstance(ip, int):
         ip = ip_interface(str(ip)).version
     return 'ipv6' if ip == 6 else 'ip'
+
+
+def family_translate(family: str):
```

Comment:
Maybe it would be better to move that to a property of AddressFamily
This would remove the need of importing the method to the template

Reply:
Changed: 1ea53a59706895d8a723d18302eedcdf4ddd2e5f